ALS/Source/ALS/Private/Notify/AlsAnimNotify_FootstepEffects.cpp

```diff
@@ -52,7 +52,7 @@ void UAlsAnimNotify_FootstepEffects::Notify(USkeletalMeshComponent* MeshComponen
 #endif
 
 	const FAlsFootstepEffectSettings* EffectSettings{nullptr};
-	const auto SurfaceType{Hit.PhysMaterial.IsValid() ? Hit.PhysMaterial->SurfaceType : SurfaceType_Default};
+	const auto SurfaceType{Hit.PhysMaterial.IsValid() ? Hit.PhysMaterial->SurfaceType : TEnumAsByte<EPhysicalSurface>(SurfaceType_Default)};
```

Comment:
I think in this case it's better to just replace `auto` with `EPhysicalSurface`.